dsacalib/weights.py

```diff
@@ -1,24 +1,26 @@
 """Determine and inspect beamformer weights."""
 
+from typing import List
 import glob
 import os
 from collections import namedtuple
+from datetime import datetime, timedelta, timezone
 
 import astropy.units as u
 import matplotlib.pyplot as plt
 import numpy as np
 import yaml
 from antpos.utils import get_itrf
 import scipy  #pylint: disable=unused-import # must come before casacore
-from casacore.tables import table
 from dsautils import cnf
 
 import dsacalib.constants as ct
 from dsacalib.fringestopping import calc_uvw
-from dsacalib.ms_io import get_antenna_gains, get_delays, read_caltable
+from dsacalib.ms_io import get_antenna_gains, get_delays, read_caltable, freq_GHz_from_ms
 
 
 def get_refmjd() -> float:
```

Comment:
Not necessary now, but let's move all of these cnf lookups to their own module at some point.